.github/workflows/release.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     - uses: actions/setup-python@v5
       with:
         python-version: '3.x'
+        cache: 'pip' # caching pip dependencies
```

Comment:
This only works if using a `requirements.txt` file as stated [here](https://github.com/actions/setup-python#caching-packages-dependencies)

```suggestion
```